raiden/ui/cli.py

```diff
@@ -584,10 +600,27 @@ def app(
     # this assumes the eth node is already online
     check_json_rpc(rpc_client)
 
+    net_id = blockchain_service.network_id
+    if isinstance(network_id, str):
+        network_id_numeric = NETWORKNAME_TO_ID[network_id]
```

Comment:
I would suggest always converting the input in `NetworkChoiceType` to `int` so that we don't have to deal with various types internally.

Although it might be nice to back-convert to human-readable names in error messages like below.